src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-8160.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2023-02-06'
 version: 8.16.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.16.0.gem'
+features: ["Add new Ruby on Rails library instrumentations"]
```

Comment:
I think this one might be more enticing if there were a few more details; however, that might get too hairy. Thoughts?

Reply:
Definitely worth it to add a little more here :) Good call! [Done](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-e5b6acef01f8897cc5bdbea96cf263627fcc52909e70c10d43ccc31003c2ecb7R6)